processor/asset/sourcemap/sourcemap.go

```diff
@@ -85,7 +82,7 @@ func (p *sourcemapProcessor) Validate(raw map[string]interface{}) error {
 
 	_, err := parser.Parse("", []byte(smap))
 	if err != nil {
-		return errors.New(fmt.Sprintf("Error validating sourcemap: %v", err))
+		return errors.Wrap(err, "error validating sourcemap")
```

Comment:
did you check how this reads? sometimes these `wrap` calls feel very unnatural and redundant, and what matters for the user at the end is that the sourcemap hasn't been generated correctly, as the user can't do nothing with a parsing error other than generating it again.

Reply:
Yeah, the error looks the same, like: `error validating sourcemap: unexpected end of JSON input`